src/latlon.jl

```diff
@@ -0,0 +1,51 @@
+import ArchGDAL
+const AG = ArchGDAL
+using Proj
+
+# px2xy function from creator of ArchGDAL https://github.com/yeesian/ArchGDAL.jl/issues/68
+# Uses the geotransform from the image to map pixels to projection values
+function px2xy(gt, x, y)
+    xoff, a, b, yoff, d, e = gt
+    xp = a * (x + 0.5) + b * (y + 0.5) + xoff
+    yp = d * (x + 0.5) + e * (y + 0.5) + yoff
+    (xp, yp)
+end
+
+"""
+Reads the GeoTiff located at <imgpath>, extracts the coordinate reference system,
+and produces a lookup table with for the column and row values in the same projection
+as the GeoTiff and a 2D array for latitude and longitude.
+"""
+
+# rewrite of the latlon.py library with Julia
+function latlon(imgpath)
+    # readraster automatically detects GeoTIFF
+    im = AG.readraster(imgpath) 
+
+    # get a reference projection (WGS84)
+    ref = AG.toPROJ4(AG.importEPSG(4326))
+
+    # get the raster projection information and format it
+    p = AG.getproj(im); 
+    crs = AG.toPROJ4(AG.importWKT(p));
+
+    nrows, ncols, nlayers = size(im)
```

Comment:
Use `_` for unused unpacked values.
```suggestion
    nrows, ncols, _ = size(im)
```